Atomic/client.py

```diff
@@ -41,20 +48,26 @@ def __getattr__(self, name):
 
     def __getattribute__(self, name):
         # Avoid recursion for self._dockerclient
-        if name == "_dockerclient":
+        if name == "_dockerclient" or name == "_client":
             return object.__getattribute__(self, name)
         obj = self._dockerclient
         attr = docker.AutoVersionClient.__getattribute__(obj, name)
         if hasattr(attr, '__call__'):
             def newfunc(*args, **kwargs):
-                result = attr(*args, **kwargs)
-                return iter_subs(result)
+                try:
+                    result = attr(*args, **kwargs)
+                    return iter_subs(result)
+                except requests.exceptions.ConnectionError as e:
```

Comment:
If the docker daemon is down, then when the containers or image initially tries to connect to the service, it will throw an exception requests.exceptions.ConnectionError, we want to return and empty list if the user was looking for containers or images.  This way atomic images list can work with system containers and not just throw an exception saying docker is not running.